doc/modules/preprocessing.rst

```diff
@@ -750,11 +751,47 @@ categories are `min_frequency` and `max_categories`.
    input feature. `max_categories` includes the feature that combines
    infrequent categories.
 
-In the following example, the categories, `'dog', 'snake'` are considered
-infrequent::
+In the following example with :class:`OrdinalEncoder`, the categories `'dog' and
+'snake'` are considered infrequent::
 
    >>> X = np.array([['dog'] * 5 + ['cat'] * 20 + ['rabbit'] * 10 +
    ...               ['snake'] * 3], dtype=object).T
+   >>> enc = preprocessing.OrdinalEncoder(min_frequency=6).fit(X)
+   >>> enc.infrequent_categories_
+   [array(['dog', 'snake'], dtype=object)]
+   >>> enc.transform(np.array([['dog'], ['cat'], ['rabbit'], ['snake']]))
+   array([[2.],
+          [0.],
+          [1.],
+          [2.]])
+
+:class:`OrdinalEncoder`'s `max_categories` do **not** take into account missing
+or unknown categories. Setting `unknown_value` or `encoded_missing_value` to an
+integer will increase the number of unique integer codes by one each. This can
+result in up to `max_categories + 2` integer codes. In the following example,
+"a" and "d" are considered infrequent and grouped together into a single
+category, "b" and "c" are their own categories, unknown values are encoded as 3
```

Comment:
Can you explain the motivation of treating unknown values different from infrequent ones? Or is a simplification for now?

Reply:
It's a simplification to get infrequent categories without adding more API. To treat unknown categories as the infrequent ones, we'll need to add a new `handle_unknown="infrequent_if_exist"`.

Reply:
ok that's fair :)